src/components/AvatarMenu/AvatarMenu.stories.tsx

```diff
@@ -0,0 +1,140 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { AvatarMenu } from './AvatarMenu';
+import { DEFAULT_AVATAR_MENU_ITEMS } from './AvatarMenu.constants';
+import SettingsIcon from '@mui/icons-material/Settings';
+import LogoutIcon from '@mui/icons-material/Logout';
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import type { AvatarMenuItem, PropsAvatarMenu } from './AvatarMenu.types';
+
+const extendedDefaultItems: AvatarMenuItem[] = DEFAULT_AVATAR_MENU_ITEMS.map((item) => ({
+	...item,
+	icon: item.label.includes('Perfil') ? (
+		<AccountCircleIcon fontSize="small" />
+	) : item.label.includes('Configuración') ? (
+		<SettingsIcon fontSize="small" />
+	) : item.label.includes('Sesión') ? (
+		<LogoutIcon fontSize="small" />
+	) : undefined,
+}));
+
+const customItems: AvatarMenuItem[] = [
+	{
+		label: 'Cambiar a Modo Oscuro',
+		onClick: () => console.log('Dark Mode'),
+		'data-testid': 'dark-mode-option',
+	},
+	{ label: 'Ayuda y Soporte', onClick: () => console.log('Help'), 'data-testid': 'help-option' },
+	{ label: 'Cerrar Sesión', onClick: () => console.log('Logout'), divider: true },
+];
+
+interface AvatarMenuStoryArgs extends PropsAvatarMenu {
+	onItemClicked: (label: string) => void;
+}
+
+const meta: Meta<AvatarMenuStoryArgs> = {
+	title: 'Navigation/AvatarMenu',
+	component: AvatarMenu,
+	parameters: {
+		layout: 'centered',
+		docs: {
+			description: {
+				component: 'Componente de menú desplegable para usuario autenticado',
+			},
+		},
```

Comment:
You use an arg named 'data-testid' (with a hyphen) in argTypes/args. Storybook args with hyphenated names are awkward and can cause problems; prefer camelCase prop names in stories/controls.

**Sugerencia:**
```typescript
Change story arg keys to camelCase (e.g. dataTestId) and map them to component props if necessary. This keeps controls simpler and consistent.
```